src/helpers/figures/charts/chart_ui_common.ts

```diff
@@ -255,6 +256,14 @@ export function getChartDatasetValues(getters: Getters, dataSets: DataSet[]): Da
       label = `${ChartTerms.Series} ${parseInt(dsIndex) + 1}`;
     }
     let data = ds.dataRange ? getData(getters, ds) : [];
+    if (
+      data.every(
+        (cell) =>
+          cell === undefined || cell === null || !isNumber(cell.toString(), getters.getLocale())
```

Comment:
doesn't this breaks what's just below where it looks like the dataset should *not* be ignored if it contains only strings.

Reply:
Yes, you're right. I'll invert the order of the tests